code/_onclick/drag_drop.dm

```diff
@@ -16,3 +16,12 @@
 // recieve a mousedrop
 /atom/proc/MouseDrop_T(atom/dropping, mob/user, src_location, over_location, src_control, over_control, params)
 	return
+
+/atom/proc/CanMouseDrop(atom/over, var/mob/user = usr, var/incapacitation_flags)
```

Comment:
Nooope nuke this proc
Massively high risk, and this will break anything in the game that relies on distant drag and drop

There's no reason to limit it to adjacent only at such a low level.

Reply:
its just a check, its dont actualy calls itself anywhere

Reply:
hmm i may have spoken too soon actually, this could just be overridden for anything that wants to do longranged drops right?

Reply:
Made it into simple proc to eliminate confusioness